typedoc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://typedoc.org/schema.json",
+  "cleanOutputDir": true,
+  "entryPoints": ["src/index.ts"],
+  "entryPointStrategy": "expand",
+  "gitRevision": "release",
+  "githubPages": false,
+  "hideGenerator": true,
+  "includeVersion": false,
+  "mergeModulesMergeMode": "project",
+  "mergeModulesRenameDefaults": true,
+  "name": "AssertiveTS - API Reference",
```

Comment:
```suggestion
  "name": "React Native Spotlight Tour - API Reference",
```
?

Reply:
Great catch! Thank you @CristhianMotoche 🎉